### PR TITLE
Allow messaging for expired memberships.

### PR DESCRIPTION
**Pull request by eschlange**
* Removed conditional logic to hide compose and reply.
* Added specs to check for visible components.

[#91437624]

### CODE REVIEW

File: app/views/think_feel_do_engine/messages/_show.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<% if can? :create, SiteMessage %>
+  <div class="btn-toolbar" role="toolbar">
+    <%= link_to "Messages", coach_group_messages_path(@group), class: "btn btn-default" %>
+    <%= link_to "Compose", new_coach_group_message_path(@group), class: "btn btn-default" %>
+  </div>
+<% end %>
+
+<% if participant_signed_in? %>
+  <p class="lead">You will be able use this tool once you have been assigned a coach</p>
+<% elsif !current_participant.nil? && current_participant.active_membership.present? && current_participant.in_study? %>
```
Comment (ericcf) [inline app/views/think_feel_do_engine/messages/_show.html.erb]: FYI, `nil` is falsy, so an alternative here could be just `elsif current_participant...`

Comment (eschlange) [inline app/views/think_feel_do_engine/messages/_show.html.erb]: Good to know! Thanks! My Java roots are strong, which is why I tend to do this sort of thing. I'll learn over time =D

**Comment by ericcf** [comment]
Is this feature being added only to TFDSo?

**Comment by eschlange** [comment]
Also implemented this in sunnyside(https://github.com/cbitstech/sunnyside/pull/18). This should not exist in stepped care.

**Pull request merged**